### PR TITLE
PMM-7 Make target to rebuild agent in devcontainer

### PR DESCRIPTION
**Pull request by michal-kralik**
PMM-7

Build: SUBMODULES-2616

### CODE REVIEW

File: Makefile.devcontainer [hunk 1]
```diff
@@ -2,22 +2,30 @@
 
 include Makefile.include
 
-release-dev:
+release-dev-managed:
```
Comment (artemgavrilov) [inline Makefile.devcontainer:5]: Can you please also add comments to all these new targets. We are using them to print output of `make help`

**Comment by michal-kralik** [comment]
Great feedback @artemgavrilov, thanks!

**Review by artemgavrilov — APPROVED** [review]

**Review by BupycHuk — APPROVED** [review]

**Comment by BupycHuk** [comment]
looks good to me, what if we run a separate pmm-client dev container?

**Comment by artemgavrilov** [comment]
> looks good to me, what if we run a separate pmm-client dev container?

Mmm, but what is the reason to do that?

**Comment by BupycHuk** [comment]
@michal-kralik let's merge this PR

**Pull request merged**